samples/cds-sample-application/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "cap-e2e-app",
+  "version": "0.0.1",
+  "description": "SAP Cloud SDK for JS - Test application for CAP",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "deploy": "npm run ci-build && npm run ci-package && cf push",
+    "ci-build": "npm run cds-deploy && npm run cds-build && npm run build",
```

Comment:
general question if we want to include the `pipeline` parts in this repo. IMO this is not part of the SDK, but perhaps you can ask this in the chat so we have a common decision.